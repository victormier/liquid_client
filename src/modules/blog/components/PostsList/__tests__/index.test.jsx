import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import { PostListComponent } from '../index'


describe('component', () => {
  const listProps = {
    items: [
      {
        id: 1,
        title: 'Item title',
        text: 'Item text',
      },
    ],
  }

  it('renders with all props', () => {
    const component = shallow(
      <PostListComponent {...listProps} />
    )

    expect(shallowToJson(component)).toMatchSnapshot()
  })
})
