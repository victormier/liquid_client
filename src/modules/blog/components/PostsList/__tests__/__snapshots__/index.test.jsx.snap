// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component renders with all props 1`] = `
<Grid
  bsClass="container"
  componentClass="div"
  fluid={false}
>
  <Row
    bsClass="row"
    componentClass="div"
  >
    <Col
      bsClass="col"
      componentClass="div"
    >
      <h3>
        Item title
      </h3>
      <p>
        Item text
      </p>
      <Link
        onlyActiveOnIndex={false}
        style={Object {}}
        to="/posts/1"
      >
        <Button
          active={false}
          block={false}
          bsClass="btn"
          bsStyle="primary"
          disabled={false}
          onMouseOver={[Function]}
        >
          Detail
        </Button>
      </Link>
    </Col>
  </Row>
</Grid>
`;
